assets/src/components/App.js

```diff
@@ -1,26 +1,36 @@
-const template = `
-<div>
-  <h1>{{ helloWorld }}</h1>
-</div>`;
-
-import zdClient from '../libs/ZDClient.js';
+import zdClient from './../libs/ZDClient.js';
+import appTemplate from './AppTemplate.js';
+import Child from './Child/Child.js';
 
 const App = {
-  template,
+  template: appTemplate,
+  components: {
+    Child
+  },
   data() {
     return {
-      fieldValue: '',
+      state: {}
     };
   },
+  created() {
+    this.state.zdInstance = zdClient.app.client._instanceGuid;
+    this.initApp();
```

Comment:
why would you call a method to init the app? how many times `initApp` will be called?